tests/test_benchmark.py

```diff
@@ -155,6 +155,7 @@ def test_inference_with_configs(self):
     def test_inference_encoder_decoder_with_configs(self):
         MODEL_ID = "sshleifer/tinier_bart"
         config = AutoConfig.from_pretrained(MODEL_ID)
+        config.use_cache = False
```

Comment:
without this line `test_inference_encoder_decoder_with_configs`  fails with `AssertionError: pass decoder_past_key_value_states to use_cache` at line 541 in `modeling_bart.py`

Reply:
Yeah, this is a hack to make the test pass - it should not be needed. Actually, I can take a look next week in combination with https://github.com/huggingface/transformers/issues/6353 -> might be related. Or do you want to take a look @sshleifer . This test fails without `config.use_cache=False`, but it should not fail. It did not fail before and is likely due to the changes done in `modeling_bart.py`